src/test/java/cloudBankPages/SystemUserListPage.java

```diff
@@ -0,0 +1,36 @@
+package cloudBankPages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import static org.testng.Assert.assertTrue;
+
+public class SystemUserListPage {
+
+    WebDriver driver;
+
+    @FindBy(xpath = "//h1[text()='System User List']")
+    public WebElement systemUserListTitle;
+
+    public WebElement getUserNameListLocator(String text){

```

Comment:
You have several methods that look exactly like this. I suggest you create another folder called "helper" a put there methods that are reusable across different classes.